dashboard/templates/blocks/header.ejs

```diff
@@ -31,52 +31,7 @@
 
 <body>
   <a name="top"></a>
-  <nav class="navbar navbar-expand-lg navbar-dark bg-primary">

```

Comment:
Not sure why you felt the need to remove all of this code, as it completely ruins the look and design.

[Currently available](https://i.imgur.com/WLJyGo3.png)
[Your proposed changes](https://i.imgur.com/uIv4YIu.png)

Reply:
All because of the container, i will change it